src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,22 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
```

Comment:
I recommend doing destructuring for the object 
`const { comments, user, title, body } = post;`